cypress/e2e/opening-hours-editor.cy.ts

```diff
@@ -366,4 +527,119 @@ describe("Opening hours editor", () => {
       endDate: oneMonthFromToday(),
     });
   });
+
+  it("Can edit all opening hours series", () => {
+    createOpeningHoursSeries({
+      openingHourCategory: OpeningHourCategories.SelfService,
+      start: "10:00",
+      end: "16:00",
+      endDate: oneMonthFromToday(),
+    });
+    updateOpeningHoursSeries({
+      openingHourCategory: OpeningHourCategories.SelfService,
+      start: "09:00",
+      end: "15:00",
+    });
+  });
+
+  type EditRestOfOpeningHoursSeriesType = {
+    editSeriesFromIndex: number;
+    openingHourCategory: OpeningHourCategories;
+    originalStart: TimeString;
+    originalEnd: TimeString;
+    updatedStart: TimeString;
+    updatedEnd: TimeString;
+  };
+
+  it("Can edit rest of opening hours series", () => {
+    const {
+      originalStart,
+      originalEnd,
+      updatedStart,
+      updatedEnd,
+      openingHourCategory,
+      editSeriesFromIndex,
+    }: EditRestOfOpeningHoursSeriesType = {
+      editSeriesFromIndex: 2,
+      openingHourCategory: OpeningHourCategories.SelfService,
+      originalStart: "10:00",
+      originalEnd: "16:00",
+      updatedStart: "09:00",
+      updatedEnd: "15:00",
+    };
```

Comment:
I find it quite hard to read this object creation/deconstruction that seems to be going on here.

Does it provide any value compared to just assigning values e.g. `const editSeriesFromIndex: int = 2` etc.? 

It also occurs a couple of other places in this PR.